hack/quickstart-setup.sh

```diff
@@ -16,12 +16,19 @@
 # limitations under the License.
 #
 
+BIN_DIR=$(pwd)/bin
```

Comment:
i wonder if we should put this as `/tmp/mgc/bin` as we don't know where the command will be run from. if a cleanup is run seperately and potenetially from a different dir we could end up not able to clean up things.